lib/promscrape/scrapework.go

```diff
@@ -877,7 +904,7 @@ func (sw *scrapeWork) addAutoTimeseries(wc *writeRequestCtx, name string, value
 	sw.addRowToTimeseries(wc, &sw.tmpRow, timestamp, false)
 }
 
-func (sw *scrapeWork) addRowToTimeseries(wc *writeRequestCtx, r *parser.Row, timestamp int64, needRelabel bool) {
+func (sw *scrapeWork) addRowToTimeseries(wc *writeRequestCtx, r *parser.Row, timestamp int64, needRelabel bool) bool {
```

Comment:
Should it be renamed to `tryAddRowToTimeseries` ?

Reply:
updated